app/controllers/bibliographic_controller.rb

```diff
@@ -295,17 +295,37 @@ def bib_id_param
       sanitize(params[:bib_id])
     end
 
+    def needs_locator_call_no(location)
+      # For now only Firestore items need this.
+      return true if (location || "").start_with?("firestone$")
+      false
+    end
+
+    def make_sortable_call_number(call_no)
+      if call_no.start_with?("Oversize ")
+        call_no.sub("Oversize", "") + " Oversize"
+      else
+        call_no
+      end
+    end
+
     def add_locator_call_no(records)
```

Comment:
The logic in this method is a bit more complicated than with Voyager because Alma inserts holdings in between and we have to loop through them.